### PR TITLE
[Web-40] empty btn coursebag

### PR DESCRIPTION
**Pull request by emilyPhee**
https://user-images.githubusercontent.com/57051508/218372513-65d43662-7296-4a1a-9a13-32ce8e6a3ce6.mov

- Add a clear button on the `CourseBag` component
- Clear `CourseBag` with confirmation modal
- Display total units when `CourseBag` has more than one item

### CODE REVIEW

**Review by emilyPhee — COMMENTED** [review]
reviewed

**Comment by emilyPhee** [comment]
[[Web] Add clear button for CourseBag](https://trello.com/c/1nK0vJYq/40-web-add-clear-button-for-coursebag)

File: new-nextjs-client/src/components/index-page/CourseBag/CourseBag.tsx [hunk 1]
```diff
@@ -50,6 +99,9 @@ function CourseBag(props: Props) {
           ))}
         </SortableContext>
       </div>
+      {appUser.courseBag.length > 0 ? (
+        <div className={cx('total-unit')}>{totalUnit} units</div>
```
Comment (anthopark) [inline new-nextjs-client/src/components/index-page/CourseBag/CourseBag.tsx]: let's do a correct pluralization here. 1 "unit", otherwise use "units"

**Review by anthopark — APPROVED** [review]
Looks good! left a small comment

**Pull request merged**